packages/snap/src/rpc/signMessage.ts

```diff
@@ -48,12 +49,14 @@ export async function signMessage(
         textAreaContent: messageCreator(
           [
             {message: 'to:', value: message.to},
-            {message: 'value:', value: message.value !== '0' && message.value},
+            {message: 'from:', value: message.from},
```

Comment:
now is data show twice
![image](https://user-images.githubusercontent.com/6376047/163369754-74da2b91-22dc-4107-b36e-3b96074a504e.png)

If we going with approach to include it in `textAreaContent` it should be be good idea to remove it from `description`

Reply:
yfi value is formatted correctly its my mistake by coping from changes :see_no_evil:

Reply:
I don't have a strong opinion on whether or not to remove from the description, but I do like the idea of showing the user _exactly_ the message they're signing (including the `from`) in the `textAreaContent`. I think it's a more secure UI practice to group the message arguments together.